scripts/apps/authoring/authoring/directives/ItemCarouselDirective.ts

```diff
@@ -79,6 +79,8 @@ export function ItemCarouselDirective(notify) {
                 .filter((key) => allowed[key] === true)
                 .map((key) => 'application/superdesk.item.' + key);
 
+            ALLOWED_TYPES.push('Files');
```

Comment:
A file is not a media type.

```
allowed: {picture: boolean, video: boolean, audio: boolean, file: boolean}
```
Having `file` inside allowed doesn't make sense. Are images, videos or audio not files? They are. Why then `files` is there? Does it support PDFs or excel files? No, it doesn't.

It's confusing for other developers and will even be confusing for you after 6 months when you forget why was it done this way.

Separate the concerns in the code. Make sure that uploading files from the file system is done in one place and when you need to add an image to the gallery, you don't need to care about where it's coming from - a file system, superdesk images. See [this diagram](https://drive.google.com/file/d/1KDzE171U79Yo7sJV75YApey-qrYcX3Ly/view?usp=sharing) I made.